src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -226,14 +264,12 @@ const styles = StyleSheet.create({
   }),
   navContainer: (isHeaderHide, top) => ({
     flexDirection: 'row',
-    height:
-      (isHeaderHide
-        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
-        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + top,
+    height: isHeaderHide
+      ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
+      : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT,
     justifyContent: 'space-between',
     alignItems: 'flex-start',
     backgroundColor: 'white',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `paddingTop` style property might affect the positioning of the nav container. Ensure that this change doesn't cause any unexpected shifts in the layout.


<!-- This is an auto-generated comment by OSS CodeRabbit -->